omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs2_0/EncounterSearchHandler2_0.java

```diff
@@ -58,7 +57,8 @@ public SearchConfig getSearchConfig() {
 	public PageableResult search(RequestContext context) throws ResponseException {
 		String patientUuid = context.getRequest().getParameter("patient");
 		String encounterTypeUuid = context.getRequest().getParameter("encounterType");
-		
+		String[] visitsUuid=context.getRequest().getParameter("visits").split(",");
```

Comment:
The name of this parameter should be singular `visit`, not plural `visits`.

Reply:
Sorry, @sadamhussain-m my bad, I think I said to use "visits", but let's go with "visit"

Reply:
okay fine,I'll modify it.